build/Makefile.kits

```diff
@@ -213,11 +229,17 @@ else # ifneq ($(COMBINED_PACKAGES),1)
 	# (Copying for combined packages)
         ifeq ($(DISABLE_LISTENER),0)
```

Comment:
Shouldn't the `DISABLE_LISTENER` stuff come out too?

I think that it shouldn't be based on installbuilder flags (like it is now), nor on the existence of a marker file to NOT enable the listener. Instead, we should never enable the listener unless a marker file is created to do so. This can be controlled by the customer (create the marker file before installing the package), or by a new qualifier to the shell-bundle (`--enable-om-port` or something).

So I'd recommend eliminating line 230, fixing indentation under it, removing definition of DISABLE_LISTENER when invoking installbuilder, then make the change to the shell bundle (and post installation scripts) appropriately.

I'll mark this waiting so I see those changes go in. Obviously, you'll need to touch the MPs on Windows to do whatever it needs (run with --enable-om-port, or create the marker file before installing the package, or whatever. That will be separate to this since that stuff still lives in TFS anyway.